File: app/src/main/java/com/example/prashant/databinding/data/ContactDatabase.java
package com.example.prashant.databinding.data;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

@Database(entities = {Contact.class}, version = 1, exportSchema = false)
public abstract class ContactDatabase extends RoomDatabase {
    public abstract ContactDao contactDao();
}

File: app/src/main/java/com/example/prashant/databinding/data/ContactDatabase.kt
package com.example.prashant.databinding.data

import android.arch.persistence.room.Database
import android.arch.persistence.room.RoomDatabase

@Database(entities = [Contact::class],
        version = 1,
        exportSchema = false)
abstract class ContactDatabase : RoomDatabase() {

    abstract fun contactDao(): ContactDao
}
